consensus_message

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 REMME
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+syntax = "proto3";
+
+message ConsensusMessagePayload {
+  // period of consensus loop
+  uint64 period = 1;
+  
+  // header_signature of block that was proposal
+  string proposal_id = 2;
+  
+  // address of node that propose this header_signature
+  string voter_id = 3;
```

Comment:
Specify the exact address format in the docstring (from which transaction family?)